draft-ietf-quic-transport.md

```diff
@@ -1213,7 +1213,7 @@ server MUST check that it would have sent a version negotiation packet if it had
 received a packet with the indicated initial_version.  If a server would have
 accepted the version included in the initial_version and the value differs from
 the value of negotiated_version, the server MUST terminate the connection with a
-QUIC_VERSION_NEGOTIATION_MISMATCH error.
+VERSION_NEGOTIATION_ERROR error.
```

Comment:
I am having some trouble understanding how to send this error https://github.com/quicwg/base-drafts/issues/597 - but it might be resolved with updated PUBLIC_RESET to replace CONNECTION_CLOSE.